src/GovUk.Education.ExploreEducationStatistics.Admin/Models/Api/MyReleaseViewModel.cs

```diff
@@ -34,6 +32,15 @@ public class MyReleaseViewModel
 
         public DateTime? Published { get; set; }
 
```

Comment:
I know it's not your change but the name of this model is really confusing! `MyReleaseViewModel`

Reply:
Yup - happy for it to get renamed to something that suggests the same meaning as this class, which is that the permissions against the Release are tailored to the individual requesting the information.  Perhaps "DashboardReleaseViewModel"?  Think it should happen outside of this PR though.